src/main/java/slimeknights/mantle/client/model/util/MantleItemLayerModel.java

```diff
@@ -96,8 +98,8 @@ public BakedModel bake(IModelConfiguration owner, ModelBakery bakery, Function<M
       builder.addAll(getQuadsForSprite(data.color(), data.noTint() ? -1 : i, sprite, transform, data.luminosity(), pixels));
     }
     // transform data
-    ImmutableMap<ItemTransforms.TransformType,Transformation> transformMap = PerspectiveMapWrapper.getTransforms(new CompositeModelState(owner.getCombinedTransform(), modelTransform));
-    return new BakedItemModel(builder.build(), particle, Maps.immutableEnumMap(transformMap), overrides, true, owner.isSideLit());
+    ImmutableMap<ItemTransforms.TransformType,Transformation> transformMap = PerspectiveMapWrapper.getTransforms(owner.getTransforms());
```

Comment:
Your code is relying on old 1.18.2 model code that doesn't exist in newer porting lib versions and was left in by mistake I suggest you look at the 1.19.4 version of this class